daemon/emer-daemon.c

```diff
@@ -1005,18 +933,8 @@ handle_login_manager_signal (GDBusProxy *dbus_proxy,
 {
   if (g_strcmp0 ("PrepareForShutdown", signal_name) == 0)
     {
-      gboolean shutting_down;
-      g_variant_get_child (parameters, 0, "b", &shutting_down);
-      if (shutting_down)
-        {
-          flush_to_persistent_cache (self);
-          update_timestamps (self);
-          release_shutdown_inhibitor (self);
-        }
-      else
-        {
-          inhibit_shutdown (self);
-        }
+      flush_to_persistent_cache (self);
```

Comment:
@wgreenberg @cosimoc is it intentional that we now flush to the persistent cache and update timestamps both when the system is shutting down and also when another process cancels shutdown? I was assuming we would only want to flush when the system is shutting down to save disk I/O on eMMC devices and SD cards.